xml/dm_registers.xml

```diff
@@ -396,7 +398,8 @@
         executed.
 
         Writing this register while an abstract command is executing causes
-        \FdmAbstractcsCmderr to be set to 1 (busy) if it is 0.
+        \FdmAbstractcsCmderr to become to 1 (busy) if it is 0, when busy
+        becomes 0.
```

Comment:
This wording seems a little awkward.  I think you mean that cmderr is set to 1 because of this write, but only after the currently-executing command finishes?  Why the delay?  I guess if the current command exits with cmderr=2, then it is changed to cmderr=1 (i.e. 1 is a higher priority in case of multiple errors)?

Reply:
The idea is to allow implementations to have cmderr contain arbitrary values while the command is executing. So we don't define what its value should be until busy becomes 0. Sounds like this isn't clear. What would be a better way to phrase this concisely?

Reply:
How about something like "Writing to this register while an abstract command is executing with cmderr=0 causes cmderr to be set to 1 (busy) at or before the conclusion of that command"?

Reply:
Here's wording that had been suggested earlier:
"Writing this register while an abstract command is executing causes cmderr to become 1 (busy) once the command completes (busy becomes 0), if cmderr was 0 when the command started."

This phrasing avoids making any assumptions about the value of cmderr while the command is executing, as Tim requested.  Is it clear?  "executing with cmderr=0" makes such an assumption.

Reply:
"if cmderr was 0 when the command started" is not necessary. If cmderr wasn't 0, then the command would not have started. So how about

"Writing this register while an abstract command is executing causes cmderr to become 1 (busy) once the command completes (busy becomes 0)."

Reply:
That sounds good to me.

Reply:
Right.  Yes, sounds good.
Couple of observations.

1. This design means there is hidden state (whether the register was written while a command is executing, is not visible until the command completes).  Seems okay, if done knowingly.
2.  The same sentence exists for abstractcs, command, and abstractauto.  Will it change for the other two.

Reply:
I'm fine with hidden state. We already don't know what is being executed, or where things are in the process.

Thanks for pointing out the other sentences. I've updated them also.